app-office/libreoffice/libreoffice-4.4.9999.ebuild

```diff
@@ -571,17 +566,17 @@ src_install() {
 	make DESTDIR="${D}" distro-pack-install -o build -o check || die
 
 	# Fix bash completion placement
-	newbashcomp "${ED}"/etc/bash_completion.d/libreoffice.sh ${PN}
+	newbashcomp "${ED}"etc/bash_completion.d/libreoffice.sh ${PN}
 	bashcomp_alias \
 		libreoffice \
 		unopkg loimpress lobase localc lodraw lomath lowriter lofromtemplate loweb loffice
-	rm -rf "${ED}"/etc/ || die
+	rm -rf "${ED}"etc/ || die
```

Comment:
yep, if you want to be future-proof, you can also do `rm -rf "${ED%/}"/etc/ || die`, but not sure if that makes sense to do on global scale